src/rust.rs

```diff
@@ -210,13 +210,24 @@ fn package_info(_workspace_root: &Utf8Path, package: &PackageMetadata) -> Result
         }
     }
 
+    let keywords_and_categories: Option<Vec<String>> =
+        if package.keywords().is_empty() && package.categories().is_empty() {
+            None
+        } else {
+            let categories = package.categories().to_vec();
+            let mut keywords = package.keywords().to_vec();
+            keywords.extend(categories);
```

Comment:
wondering if we need to do any deduping here- do people use keywords that are identical to categories?